Source/WebCore/page/scrolling/mac/ScrollingTreeScrollingNodeDelegateMac.mm

```diff
@@ -68,6 +68,10 @@
         }
     }
     
+    if (scrollingStateNode.hasChangedProperty(ScrollingStateNode::Property::MouseDidEnterScrollbar)) {
+        scrollingStateNode.mouseDidEnterScrollbar() ? m_scrollerPair->mouseEnteredScrollbar() : m_scrollerPair->mouseExitedScrollbar();
```

Comment:
/* Unofficial Review */
No return value is taken. Please consider if else instead.

Reply:
This part changed under the new patch.